src/containers/BooksForm.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+const BooksForm = () => {
+  const categories = [
+    'Action',
+    'Biography',
+    'History',
+    'Horror',
+    'Kids',
+    'Learning',
+    'Sci-Fi',
+  ];
+  return (
+    <form className="book-form" action="#">
+      <h3>
+        Create new book
+      </h3>
+      <br />
+      <div className="form-fields">
+        <label htmlFor="title">
+          Book Title:
+
+          <input type="text" id="title" name="title" />
+        </label>
+
+        <label htmlFor="category">
+          Category:
+          <select name="category" id="category">
+            {
+                    categories.map((cat, id) => (
+                      <option key={`opt_${id * 2}`} value={id}>{cat}</option>
+                    ))
+                }
+          </select>
+        </label>
+
+        <button type="button">Add Book</button>
```

Comment:
- [ ] it renders form with title input, categories select box and submit [screenshot from pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone2/Form/RenderForm.png) (This button needs to be of type submit to send all the form information and to be able to create the book, right now it is just a button with a label that doesn't serve a purpose to the form, please fix it to comply with the project requirements) (Form does not need to be functional but it should comply with the submit type for now)

Reply:
Converted button type to "submit".